python/paddle/fluid/layers/nn.py

```diff
@@ -7580,3 +7581,74 @@ def hash(input, hash_size, num_hash=1, name=None):
         attrs={'num_hash': num_hash,
                'mod_by': hash_size})
     return out
+
+
+@templatedoc()
+def grid_sampler(x, grid, name=None):
+    """
+    It sample input X by grid gennerate by AffineGridOp. The grid of shape
```

Comment:
This operation samples input X by using  bilinear interpolation based on flow field grid, which is usually gennerated by `affine_grid`.